admin/app/components/solidus_admin/ui/toast/component.rb

```diff
@@ -3,7 +3,7 @@
 class SolidusAdmin::UI::Toast::Component < SolidusAdmin::BaseComponent
   SCHEMES = {
     default: %w[
-      bg-gray-800 text-white
+      bg-fullBlack text-white
```

Comment:
@rainerdema which naming convention do we want to use for color variables? 
Probably we can safely use hyphens instead of camelCase. I don't think there'll be another fullBlack i.e. `fullBlack-100`. What are your thoughts about it?

Reply:
Hey @mfrecchiami, I initially used camelCase for 'fullBlack' because I noticed other similar colors in the Tailwind configuration were formatted this way (likely to avoid the need for wrapping them in quotes 🤔).
However, based on your feedback, I've now changed all the colors to kebab-case in the latest commit.
Let me know WDYT 👍

Reply:
Awesome! Yes, Tailwind uses the camelCase format but the output is a classname in kebab-case. Thanks for this update ❤️